base/range.jl

```diff
@@ -54,10 +54,7 @@ optionally with a given step (defaults to 1, a [`UnitRange`](@ref)).
 One of `length` or `stop` is required.  If `length`, `stop`, and `step` are all specified, they must agree.
 
 If `length` and `stop` are provided and `step` is not, the step size will be computed
-automatically such that there are `length` linearly spaced elements in the range (a [`LinRange`](@ref)).
-
-If `step` and `stop` are provided and `length` is not, the overall range length will be computed
```

Comment:
Hmm, maybe I didn't notice this before or maybe it changed, but now you've deleted explanation of what happens if you don't provide `length`. This does need to be explained...

Reply:
Yes, It was me who did not add it earlier as well. Edited it now. Thanks